js/nova.js

```diff
@@ -298,6 +298,7 @@ module.exports = class nova extends Exchange {
             'currency': code,
             'amount': amount,
             'address': address,
+            'addressTo': address,
```

Comment:
This OK since we know that both deposit and withdrawal addresses is receiving address?

Reply:
Yep, thx)